File: UI/src/main/java/ca/warp7/rt/feature/predictor/PredictorFeature.java
package ca.warp7.rt.feature.predictor;

import ca.warp7.rt.core.feature.Feature;
import ca.warp7.rt.core.feature.FeatureItemTab;

import java.util.Collections;
import java.util.List;

public class PredictorFeature implements Feature {
    @Override
    public List<FeatureItemTab> getLoadedTabs() {
        return Collections.singletonList(new FeatureItemTab(
                "Match Predictor", "fas-balance-scale:16:gray", getFeatureId(),
                FeatureItemTab.Group.SingleTab, ""
        ));
    }

    @Override
    public String getFeatureId() {
        return "predictor";
    }
}

File: UI/src/main/kotlin/ca/warp7/rt/feature/predictor/PredictorFeature.kt
package ca.warp7.rt.feature.predictor

import ca.warp7.rt.core.feature.Feature
import ca.warp7.rt.core.feature.FeatureItemTab

class PredictorFeature : Feature {
    override fun getLoadedTabs(): List<FeatureItemTab> {
        return listOf(FeatureItemTab(
                "Match Predictor", "fas-balance-scale:16:gray", featureId,
                FeatureItemTab.Group.SingleTab, ""
        ))
    }

    override fun getFeatureId(): String {
        return "predictor"
    }
}
